spec/features/admin_claim_without_personal_information_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "rails_helper"
+
+RSpec.feature "Admin claims with personal information removed" do
```

Comment:
We [want to move](https://trello.com/c/qHKsfiem/1367-rename-pii-to-personal-data) to using the term "personal data" instead of "PII". So perhaps it would be best to use "personal data" instead of "personal information" here and in the filename?

Reply:
Oops, good spot, that's what I thought I was doing but clearly didn't read the card and used the wrong phrasing!